File: ui-languageMentor/src/app/examinations/shared/models/point-model.ts
import { AnswerModel } from './answer-model';

export class PointModel {
    public PointId: number;
    public PointText: string;
    public AnswerChoices: AnswerModel[];
    public SelectedAnswers: AnswerModel[];
    public IsMultipleChoices: boolean;
}

File: LanguageMentor.Web.Api/Models/PointModel.cs
using System.Collections.Generic;

namespace LanguageMentor.Web.Api.Models
{
    public class PointModel
    {
        public int PointId { get; set; }

        public string PointText { get; set; }

        public int ExerciseId { get; set; }

        public IList<AnswerModel> AnswerChoices { get; set; }

        public IList<AnswerModel> SelectedAnswers { get; set; }

        public bool IsMultipleChoices { get; set; }
    }
}